etcdserver/etcdhttp/client.go

```diff
@@ -539,6 +539,19 @@ func parseKeyRequest(r *http.Request, clock clockwork.Clock) (etcdserverpb.Reque
 		pe = &bv
 	}
 
+	// refresh is nullable, so leave it null if not specified
+	var refresh *bool
```

Comment:
it is probably better to check the refresh and value co-exist case here instead of in store pkg?